templates/headless/sshnpd.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#v1.0.0
+# allow machine to bring up network
+sleep 10
+USER=$(whoami)
+export USER
+while true
```

Comment:
loops [should also be indented](https://google.github.io/styleguide/shellguide.html#s5.4-loops)